app/src/main/java/com/andrew/tasky/presentation/agenda/AgendaFragment.kt

```diff
@@ -41,22 +42,21 @@ class AgendaFragment : Fragment(R.layout.fragment_agenda) {
         setOnClickListeners()
         setupCurrentMonthTextView()
         setupAgendaItemListRecyclerView()
-    }
-
-    private fun subscribeToObservables() {
+        miniCalendarAdapter = MiniCalendarAdapter(
+            onHolderClick = { date -> viewModel.setDateSelected(date) }
```

Comment:
Would rather call this `onDateClick` since from the outside it's not clear what the `holder` is